### PR TITLE
Update examples to use load_sample

### PR DESCRIPTION
**Pull request by matthewturk**
This updates the examples to use load_sample.  Once #91 is included it'll be *just* those changes.

### CODE REVIEW

**Comment by neutrinoceros** [comment]
wait the images are still shown as changed

**Comment by matthewturk** [comment]
a merge fixed it :)

**Review by neutrinoceros — APPROVED** [review]

File: examples.yaml [hunk 1]
```diff
@@ -93,7 +93,7 @@
       id: extrema
       content: |
         import yt
-        ds = yt.load("Enzo_64/DD0043/data0043")
+        ds = yt.load_sample("Enzo_64", "DD0043/data0043")
```
Comment (neutrinoceros) [inline examples.yaml:96]: With the current version of load_sample, this is how you need to write this :)
```suggestion
        ds = yt.load_sample("Enzo_64/DD0043/data0043")
```

**Review by neutrinoceros — CHANGES_REQUESTED** [review]
One case needs to be updated, the rest of them are still valid !

**Comment by neutrinoceros** [comment]
Btw you may prefer to keep the full paths in every example. It's not _required_, but it produces identical results and may be considered best practice (depending on the example we want to give). Normally, the whole patch could just be about substituting `yt.load(` with `yt.load_sample(`, and if we don't remove the full path data, the reversed substitution will also work as long as the data is already installed (as was assumed in existing versions of the docs).

**Comment by matthewturk** [comment]
Hey, what do we think about this nowadays?

File: examples.yaml (hunk 1 above)
Comment (neutrinoceros) [reply examples.yaml:96]: still relevant

Comment (matthewturk) [reply examples.yaml:96]: Right -- this *used* to work, and no longer does.

**Comment by neutrinoceros** [comment]
it's in better shape now that `yt.load_sample` has been shipped for 4 releases over a year. However the function still requires two optional deps (pooch and pandas), and personally I don't like it that we force users to install pandas just for this (for the record it was *my* idea from 2 years ago), so I would like to refactor it so pandas isn't needed, but I digress.

We wrap `ImportError` for both these dependencies so it limits the risk of confusion, so I wouldn't oppose this change now. Tell me if you want to add to it, otherwise I can just merge as is

**Comment by chrishavlin** [comment]
@neutrinoceros looks like as of the last comment you're OK with this PR, that still true? @matthewturk how you feeling about it?

**Comment by chrishavlin** [comment]
for what it's worth, +1 for being merged as-is at this point.

**Comment by neutrinoceros** [comment]
yes, go ahead. I never found the motivation to refactor pandas out of the function and at this point I doubt I ever will.

**Review by chrishavlin — APPROVED** [review]

**Pull request merged**